content/docs/guides/mediawiki-tabular-data.md

```diff
@@ -0,0 +1,61 @@
+    <td>Jakob Voß</td>
+  </tr>
+</table>
+
+[MediaWiki](https://www.mediawiki.org/) is the software used to run Wikipedia and related projects of the Wikimedia Foundation, including the media file repository [Wikimedia Commons](https://commons.wikimedia.org/). Commons hosts mostly images but also some records with tabular data. The [MediaWiki Tabular Data Model](https://www.mediawiki.org/wiki/Help:Tabular_data) was inspired by Data Package version 1 but it slightly differs from current Data Package specification, as described below.
+
+## Property Comparison
+
+A [MediaWiki tabular data page](https://www.mediawiki.org/wiki/Help:Tabular_data) describes and contains an individual table of data similar to a [Data Resource](/standard/data-resource/) with inline tabular data. Both are serialized as JSON objects, but the former comes as a page with unique name in a MediaWiki instance (such as Wikimedia Commons).
+
+### Top-level Properties
+
+MediaWiki Tabular Data has three required and two optional top-level properties. Most of these properties map to corresponding properties of a Data Resource:
+
+| MediaWiki Tabular Data                                                                                                                     | Data Package Table Schema                                                             |
+| ------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------- |
+| - (implied by page name)                                                                                                                   | [name](/standard/data-resource/#name) (required) is a string                          |
+| [description](https://www.mediawiki.org/wiki/Help:Tabular_data#Top-level_fields) (optional) is a localized string                          | [description](/standard/data-resource/#description) (optional) is a CommonMark string |
+| [data](https://www.mediawiki.org/wiki/Help:Tabular_data#Top-level_fields) (required)                                                       | [data](/standard/data-resource/#name) (optional)                                      |
+| [license](https://www.mediawiki.org/wiki/Help:Tabular_data#Top-level_fields) (required) is the string `CC-1.0` or another known identifier | [licenses](/standard/data-resource/#licenses) (optional) is an array                  |
+| [schema](https://www.mediawiki.org/wiki/Help:Tabular_data#Top-level_fields) (required) as [described below](#schema-properties)            | [schema](/standard/data-resource/#schema) (optional) can have multiple forms          |
+| [sources](https://www.mediawiki.org/wiki/Help:Tabular_data#Top-level_fields) (optional) is a string with Wiki markup                       | [sources](/standard/data-resource/#sources) (optional) is a an array of objects       |
+
+The differences are:
```

Comment:
This paragraph repeats what is listed in the table above, so I think it can be removed (easier to maintain too). If you want to keep it, that it is fine too.